CompanyAddress.md

```diff
@@ -0,0 +1,130 @@
+Company Address Service
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Sample response](#sample-response)
+- [Versioning](#versioning)
+
+## Summary
+
+Company Address Service (CAS) takes a request containing a company name, latitude and longitude and returns a geocoded response.
+
+## Request Structure
+```json
+{
+	"latitude": 59.439675,
+	"longitude": 24.746831,
+	"company_name": "Texas Honky Tonk & Cantina"
+}
+```
+
+### Response Structure
+
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node containing the following elements:
+
+| Field    | Description |
+|----------|-------------|
+| results  | A JSON array of one or more objects containing geography data. If the request returns no results, the array will be empty. |
+| expiration_date | A date string in YYYY-MM-DD format representing the last day that this data may be used. Per our contract with Google, any data cached by users of the this service must be discarded by this date.
+
+&nbsp;
+
+Each element of the returned **results** array will be formatted as follows:
+
+|   |   |
+|---|---|
+| admin_areas | A JSON array of up to 5 admin_area objects. See below for details on the structure of these objects. **This property is always present, but may be empty** if no admin areas were found. |
+| latitude | A double value specifying the location's latitude (in degrees) within the range [-90, +90]. **This property is always present.** |
+| longitude | A double value specifying the location's longitude (in degrees) within the range [-180, +180]. **This property is always present.** |
+| country | The country or region name of an address. **This property may be absent** if the location was not associated with a country. Some locations for which this occurs are "Europe", "Pacific Ocean", and "Gaza Strip" (a contested territory). |
```

Comment:
should this note still be present in this service's documentation? I mean, is there any conceivable company address request that would yield a result in the Gaza Strip?